packages/components/breadcrumb/src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'manager-breadcrumb',
+  styleUrl: 'breadcrumb.css',
+  shadow: true
+})
+export class Breadcrumb {
+  @Prop({ reflect: true }) elements: any[] = [];
```

Comment:
```suggestion
  @Prop({ reflect: true }) elements: BreadcrumbElement[] = [];
```

with 
```typescript 
interface BreadcrumbElement {
    id: number;
    url: string;
    name: string;
    active: boolean;
}
```

Even if it's not exported, it could help to self-document the `elements` property.

Reply:
@cbourgois I've updated with your suggestion